src/attributes.ts

```diff
@@ -625,6 +625,10 @@ const ATTRIBUTES_DEFINITION_BASE = {
 		name: 'Ankle_cuffs_chainable',
 		description: 'A cuff that is chainable to another item',
 	},
```

Comment:
We also need another new attribute `Hand_item` that should be added to the nails and both mittens (and later the gloves).

This can then be used by items like the binder and strait jacket to cover this attribute, making those items unable to be removed while they are under those items.

Reply:
Good point.